src/rust-crypto/rust-crypto.ts

```diff
@@ -1641,6 +1654,63 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
     public async getOwnIdentity(): Promise<RustSdkCryptoJs.OwnUserIdentity | undefined> {
         return await this.olmMachine.getIdentity(new RustSdkCryptoJs.UserId(this.userId));
     }
+
+    /**
+     * Returns whether dehydrated devices are supported by the crypto backend
+     * and by the server.
+     */
```

Comment:
For all of these methods: we can just write:

```suggestion
    /**
     * Implementation of {@link CryptoApi#isDehydrationSupported}.
     */
```

... which saves duplicating the documentation and reinforces the point that we are implementing the interface.